stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/images/CoilStreamImageLoader.kt

```diff
@@ -82,6 +95,59 @@ internal object CoilStreamImageLoader : StreamImageLoader {
         }
     }
 
+    override suspend fun loadAndResize(
+        target: ImageView,
+        data: Any?,
+        container: ViewGroup,
+        placeholderDrawable: Drawable?,
+        maxHeight: Int,
+        transformation: StreamImageLoader.ImageTransformation,
+        onStart: () -> Unit,
+        onComplete: () -> Unit,
+    ) {
+        val context = target.context
```

Comment:
Can we use here `GiphyInfo.width` and `GiphyInfo.height` to resize the container while the giphy is still loading?

Reply:
Removed these checks overall here. This now _doesn't_ resize the container per se, but applying it as a drawable that's set on the container does have different aspects of scaling as compared to the regular load.

Don't ask why (I don't know).